src/Symfony/Component/Config/FileLocator.php

```diff
@@ -42,11 +42,15 @@ public function locate($name, $currentPath = null, $first = true)
         }
 
         if ($this->isAbsolutePath($name)) {
-            if (!file_exists($name)) {
+            if (array() === $files = glob($name, GLOB_BRACE)) {
```

Comment:
missing check for GLOB_BRACE, because it's is not always defined (see php.net/glob )